include/OpenSpaceToolkit/Astrodynamics/EventCondition/Conjunctive.hpp

```diff
@@ -0,0 +1,71 @@
+/// Apache License 2.0
```

Comment:
I totally agree with being explicit and using the formal names `Conjunctive` and `Disjunctive`. However, in the future I think it would also be good to define an alias:
`using And = Conjunctive`
`using Or = Disjunctive`

Just to ease the burden for newcomers.

Reply:
Or if we could work this into defining operators on the Event Condition class that would be 🤯

Reply:
Oh wow that's an interesting idea. Let me noodle with that.

Reply:
Okay I couldn't get this to work with operators :(
Too much shared pointer cloning weirdness. I agree that having a better interface here would be nice...

Reply:
Ah well. Not that big of a deal.